pbspro.spec

```diff
@@ -137,16 +137,12 @@ Requires: tcl
 Requires: tk
 %if %{defined suse_version}
 Requires: smtp_daemon
-%if %{suse_version} >= 1500
-Requires: libical3
-Requires: hostname
-%else
-Requires: libical1
-%endif
 %else
 Requires: smtpdaemon
 Requires: libical
 %endif
+Requires: hostname
+Requires: libical
```

Comment:
You can remove the libical requirement three lines above since this one is not conditional on OS.

Reply:
@mike0042 yes i did notice that. Since this was a cherry-pick i left it as is since it is a maintenance branch and rather fix it in master? While duplicate, it seems harmless, right? 
I will still fix it if you think it is necessary.

Reply:
Not necessary. A duplicate entry should not cause a problem.

Reply:
Since this is going into 18.x I don't have a problem with it.